web/api/netdata-swagger.json

```diff
@@ -1224,12 +1242,158 @@
               "$ref": "#/definitions/chart_variables"
             }
           }
-        },
-        "result": {
-          "description": "The result requested, in the format requested."
         }
       }
     },
+    "data_json": {
+      "description": "Data response in json format",
+      "allOf": [
+        {
+          "$ref": "#/definitions/data"
+        },
+        {
+          "properties": {
+            "result": {
+              "type": "object",
+              "properties": {
+                "labels": {
+                  "description": "The dimensions retrieved from the chart.",
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  }
+                },
+                "data": {
+                  "description": "The data requested, one element per sample with each element containing the values of the dimensions described in the labels value. ",
+                  "type": "array",
+                  "items": {
+                    "type": "number"
+                  }
+                }
+              },
+              "description": "The result requested, in the format requested."
+            }
+          }
+        }
+      ]
+    },
+    "data_flat": {
+      "description": "Data response in csv / tsv / tsv-excel / ssv / ssv-comma / markdown / html formats",
+      "allOf": [
+        {
+          "$ref": "#/definitions/data"
+        },
+        {
+          "properties": {
+            "result": {
+              "type": "string"
+            }
+          }
+        }
+      ]
+    },
+    "data_array": {
+      "description": "Data response in array format",
+      "allOf": [
+        {
+          "$ref": "#/definitions/data"
+        },
+        {
+          "properties": {
+            "result": {
+              "type": "array",
+              "items": {
+                "type": "number"
+              }
+            }
+          }
+        }
+      ]
+    },
+    "data_csvjsonarray": {
+      "description": "Data response in csvjsonarray format",
+      "allOf": [
+        {
+          "$ref": "#/definitions/data"
+        },
+        {
+          "properties": {
+            "result": {
+              "description": "The first inner array contains strings showing the labels of each column, each subsequent array contains the values for each point in time",
+              "type": "array",
+              "items": {
+                "type": "array",
+                "items": {}
```

Comment:
Is this a missing pair of `{}`? For all the previous you wrote something in front of every key.

Reply:
I think this is here because the "items" is mandatory next to an array, but the contents of the array are untyped. The type changes between `string` and `number` after the first iteration. Maybe it looks strange because I squashed together the empty {} as a value?